packages/web-components/src/components/textarea/textarea.stories.ts

```diff
@@ -200,19 +202,93 @@ export const Skeleton = {
 };
 
 export const WithAILabel = {
-  render: () => html`
-    <cds-textarea label="Text Area label" helper-text="Optional helper text">
+  args,
```

Comment:
```suggestion
  args: { ...args, helperText: 'Optional helper text' },
```

Lets sync the helperText with the React story.